src/amo/sagas/reviews.js

```diff
@@ -298,7 +298,7 @@ function* deleteAddonReview({
       reviewId,
     });
 
-    yield put(unloadAddonReviews({ reviewId }));
+    yield put(unloadAddonReviews({ addonId, reviewId }));
     if (isReplyToReviewId) {
       yield put(unloadAddonReviews({ reviewId: isReplyToReviewId }));
```

Comment:
Why aren't you passing `addonId` here, too? This would also let you make `addonId` required which I think would help make the reducer a little more robust.

Reply:
I did this on purpose because we've already cleared the add-on's data in the call to `yield put(unloadAddonReviews({ addonId, reviewId }));` so there's no reason to clear it again. I was trying to be efficient.

I can see your reasoning, though. With it being optional a user might call `unloadAddonReviews` without it not realizing they they should be passing it in in order to unload the addon `too`. I'll change this.